### DISCUSSION TITLE
Package Graduation

### DISCUSSION THREAD

**Discussion by paulofaria** [GraphQLSwift/Graphiti#123] [General]
Hi, @NeedleInAJayStack! Joannis, from  SSWG reached out about switching the package to graduated status within SSWG. Here are the requirements:

## Graduation Requirements

Minimal Requirements
Have stable API (no pending/planned breaking API changes), with at least one published major release (e.g. 1.0.0)
Support new GA versions of Swift within 30d
CI setup for two latest Swift.org recommended versions of Swift
CI setup for at least one of Swift.org recommended Linux distributions
CI setup for each platform supported by the library or tool
Unit tests for both macOS and Linux
Use Swift.org docker images, when appropriate
Documented release methodology
Documented support strategy for at least one previous major version.
Explicitly define a project governance and committer process, ideally laid out in a GOVERNANCE.md file and OWNERS.md files respectively
Include list of adopters for at least the primary repo ideally laid out in an ADOPTERS.md files or logos on the project website
Optionally, have a Developer Certificate of Origin or a Contributor License Agreement

I believe we adhere to all requirements, right?

**Comment by NeedleInAJayStack** [comment]
Hey Paulo! Yeah, I think we meet almost all of those requirements with maybe the exception of a Governance doc and Adopters doc. It should be a pretty small lift to get those in.

I'm on vacation for the next week or so but could help out when I'm back. Or if you have some availability, feel free to jump on it in the meantime! Great to hear from you!

**Comment by adam-fowler** [comment]
Do you guys have a known list of adopters?

**Reply by NeedleInAJayStack** [reply]
Not a very complete one. We are using it in a commercial product at PassiveLogic, and would be happy to be on a public list of adopters. @GraphQLSwift/maintainers do you guys know of other adopters that we could list?
